build/yaml/crd/nsx.vmware.com_subnets.yaml

```diff
@@ -56,11 +56,13 @@ spec:
                     type: boolean
                 type: object
               accessMode:
+                default: Private
```

Comment:
Do we set default here or leverage the default value from NetworkConfiguration CR? @lxiaopei

Reply:
we only have podSubnetAccessmode in NetworkConfiguration CR.
For Subnet, I think we do not have default access mode, user needs to set the access mode.

Reply:
according to NSX API doc, if no access mode, then Private access mode will be used.

Reply:
updated.